src/CalcManager/UnitConverter.cpp

```diff
@@ -192,41 +191,51 @@ void UnitConverter::SwitchActive(const wstring& newValue)
     }
 }
 
-wstring UnitConverter::CategoryToString(const Category& c, const wchar_t* delimiter)
+wstring UnitConverter::CategoryToString(const Category& c, wstring_view delimiter)
 {
-    wstringstream out(wstringstream::out);
-    out << Quote(std::to_wstring(c.id)) << delimiter << Quote(std::to_wstring(c.supportsNegative)) << delimiter << Quote(c.name) << delimiter;
-    return out.str();
+    return Quote(std::to_wstring(c.id))
+        .append(delimiter)
```

Comment:
I would have preferred to use `operator+` here, but there is not an overload between `wstring` and `wstring_view`.  However, `append` will keep growing the initial string returned by the first call to `Quote` as appropriate.  Since the size of a `wstring_view` is already known, the call to append will be more efficient because the `wstring` knows the resulting string size and knows how many characters to copy upfront.